src/test/test-capability.c

```diff
@@ -20,8 +20,13 @@
 static uid_t test_uid = -1;
 static gid_t test_gid = -1;
 
-/* We keep CAP_DAC_OVERRIDE to avoid errors with gcov when doing test coverage */
-static uint64_t test_flags = 1ULL << CAP_DAC_OVERRIDE;
+#ifdef __SANITIZE_ADDRESS__
+        /* Keep CAP_SYS_PTRACE when running under Address Sanitizer */
+        static uint64_t test_flags = 1ULL << CAP_SYS_PTRACE;
+#else
+        /* We keep CAP_DAC_OVERRIDE to avoid errors with gcov when doing test coverage */
+        static uint64_t test_flags = 1ULL << CAP_DAC_OVERRIDE;
```

Comment:
while you are at it, could you change `1ULL` → `UINT64_C(1)`, please? While this is effectively the same, it's a bit less sloppy since uint64_t is effectively identical to unsigned long long, but not theoretically...

Reply:
also, could you make this `const` please? this was missing for some reason, and should really be added.

Reply:
sure, fixed